src/index.js

```diff
@@ -4,7 +4,7 @@
  * See the accompanying LICENSE file for terms.
  */
 
-import * as p from 'path';
```

Comment:
Issue seems to be here where upath is clearly used in library code, but not added to normal dependencies but only devDependencies.